engine/render/renderer/pass/RenderPass.cpp

```diff
@@ -13,7 +13,7 @@ namespace Siege
 
 RenderPass::~RenderPass()
 {
-    DestroyRenderPass();
+    // DestroyRenderPass();
```

Comment:
leftover?

Reply:
@Raelr was this intentional?

Reply:
Yes. This was intentional since I wasn't able to find a way to get this working correctly using RAII.

Reply:
Good idea to either remove or comment this kind of thing in future. Commented out code gets stale and everyone forgets what it was doing there.

Reply:
This is still not TODO'd

Reply:
Done. I've gone ahead and have implemented a move constructor to let us avoid the issue of copying with Vulkan.